sklearn/linear_model/_coordinate_descent.py

```diff
@@ -1459,6 +1459,29 @@ def _path_residuals(
 class LinearModelCV(MultiOutputMixin, LinearModel, ABC):
     """Base class for iterative model fitting along a regularization path."""
 
+    _parameter_constraints = {
+        "eps": [Interval(Real, 0, None, closed="left")],
```

Comment:
Nice catch! I would like to know how you spotted this, was it from a failing unit test ?

Reply:
Actually it's because I'm very paranoid about things like `eps, tol, max_iter` etc. being 0 since it can result in weird numerical issues (depending on how they're used). For these validation PRs I always test the estimators in a jupyter cell and play around with the values to see if there's any undocumented edge cases (e.g. here)